### PR TITLE
Setting up Git: added email details in Step 2.3

### PR DESCRIPTION
**Pull request by robertung88**
Added details regarding the email to enter in Step 2.3: Create an SSH Key so that users know to enter the private GitHub email address as well if they previously opted to use the private GitHub email.

## Because
in "Step 2.3: Create an SSH key" the user is asked to enter their email address as part of creating a new SSH key. However, the example provided is simply "your@email.com" whereas earlier in "Step 2.2: Setup Git" there was a clear callout regarding private GitHub email address, so for newcomers, the details provided in step 2.3 could be clearer in terms of what email address to enter.

## This PR

- adds more details in regards to the email to enter in Step 2.3, specifically in regards to the private GitHub email address if they previously opted to use that.

## Issue

No known issue - I could log a new issue if it's necessary.

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by fabulousgk** [comment]
The email you enter has no impact on the setup in any way.  the -c flag means comment, you are simply adding a comment to the ssh key.

**Comment by fabulousgk** [comment]
What matters is what you put in your git configuration

**Comment by robertung88** [comment]
Re-reading the step and your comment, then that's fair. It's just that as a user setting up Git for the first time, the initial callout about email address to use in Step 2.2 can still be fresh in their mind, so when asking to enter the email address in Step 2.3 can give them pause and uncertainty as to what to enter. Maybe this part below could be made better:

> The convention is to use your email as a comment to indicate who generated the public key. For example if your email address is odin@theodinproject.com, then you would type ssh-keygen -t ed25519 -C "odin@theodinproject.com".

File: foundations/installations/setting_up_git.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ ssh-keygen -t ed25519 -C "your@email.com"
 ```
 
 <div class="lesson-note lesson-note--tip" markdown="1">
-The `-C` flag is to write a comment, otherwise the key will be generated with your computer's username. The convention is to use your email as a comment to indicate who generated the public key. For example if your email address is `odin@theodinproject.com`, then you would type `ssh-keygen -t ed25519 -C "odin@theodinproject.com"`.
+The `-C` flag is to write a comment, otherwise the key will be generated with your computer's username. The convention is to use your email as a comment to indicate who generated the public key. For example if your email address is `odin@theodinproject.com`, then you would type `ssh-keygen -t ed25519 -C "odin@theodinproject.com"`. Like before, if you opted to use the private GitHub email address then you should use the private email address GitHub provided you.
```
Comment (thatblindgeye) [inline foundations/installations/setting_up_git.md:185]: Would simply stating something along the lines of, "You should use the same email that you used to initially setup Git" suffice?

Comment (thatblindgeye) [reply foundations/installations/setting_up_git.md:185]: @robertung88 checking if you've seen the comment above

**Comment by thatblindgeye** [comment]
Doesn't seem like this update is totally necessary, so going to close this out.

**Pull request closed without merging**